core/src/main/proto/spine/core/command.proto

```diff
@@ -180,3 +179,13 @@ enum CommandStatus {
     // A command was rejected during processing. No events were generated.
     REJECTED = 5;
 }
+
+// Data from a `Command` which is accessible from the code to which the command was dispatched.
```

Comment:
We need to mention that we mean consumer codebase, not the Spine's internal `...Endpoint` code or `...Repository` code. E.g.

```
 Data from a `Command` which was accessible from the command handler method to which it was dispatched.
```